test/_interface/test_amber_interface.py

```diff
@@ -722,6 +726,129 @@ def test_parse_md_config_dict_to_raw_wo_cons():
             "if_report" : True,
             "record_period" : 0.0004, # ns
             "mdstep_dir" : "./MD",
+            "use_qmmm" : False,
+            "qm_region" : None,
+            "qm_region_charge_spin" : None,
+            "qm_level_of_theory" : None,
+            "qm_engine" : None,
+            "qm_region_pdb_path" : None,
+            "qm_ele_cutoff" : None,
+            "qm_ewald" : None,
+            "qm_adjust_q" : None,
+            "qm_adaptive_solvent_type" : None,
+            "qm_num_adaptive_solvent" : None,
+            "qm_num_transition_solvent" : None,
+            "available_cores" : 9,
+            "available_mem_per_core" : "2GB",
+    }
+
+    ai = interface.amber
+    test_raw_dict = ai._parse_md_config_dict_to_raw(test_md_config_dict)
+    assert test_raw_dict == answer_raw_dict
+
+
+def test_parse_md_config_dict_to_raw_qmmm():
```

Comment:
When run on ACCRE, this test fails, as the `answer_raw_dict['namelists'][2]['method']` is `PBE0`, whereas the `tesst_raw_dict['namelists'][2]['method']` is `pbe1pbe`